### DISCUSSION TITLE
Create Standing order in Firefly

### DISCUSSION THREAD

**Discussion by masch93** [iotaledger/firefly#1658] [Help & Support]
Hello community my question is,
Is it possible to create a standing order within the Firefly app?
For example:
I will send every month 10 MI to my Daughter's wallet automatically
Thanks

**Comment by rafaeldjpbrochado** [comment]
I don't think that's possible today. Recurring transfers are a good suggestion for a future feature though. 👍 Although I'm not sure how it would work technically. Smart contracts, perhaps?
